katas/content/qft/binary_fraction_classical/SolutionB.qs

```diff
@@ -0,0 +1,9 @@
+namespace Kata {
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+
+    operation BinaryFractionClassical(q : Qubit, j : Bool[]) : Unit is Adj + Ctl {
+        R1(2.0 * PI() * IntAsDouble(BoolArrayAsInt(Reversed(j))) / IntAsDouble(1 <<< Length(j)), q);
```

Comment:
nit: even when there is some description of this solution, it might be good to assign both `IntAsDouble` expressions to variables and explain what they represent.